stdlib/docs/secp256k1_base_field_math.md

```diff
@@ -0,0 +1,10 @@
+
+## std::math::secp256k1_base_field
+| Procedure | Description |
+| ----------- | ------------- |
+| mul | Given two 256 -bit numbers ( elements belonging to secp256k1 base field ) on stack,<br /><br />where each number is represented in radix-2^32 form ( i.e. each number having eight<br /><br />32 -bit limbs ), following function computes modular multiplication of those two<br /><br />operands, computing 256 -bit result, which belongs to secp256k1 base field.<br /><br />Stack expected as below, holding input<br /><br />[a0, a1, a2, a3, a4, a5, a6, a7, b0, b1, b2, b3, b4, b5, b6, b7, ...] \| a[0..8], b[0..8] are 256 -bit numbers<br /><br />After finishing execution of this function, stack should look like<br /><br />[c0, c1, c2, c3, c4, c5, c6, c7, ...] \| c[0..8] is a 256 -bit number<br /><br />Note, for computing modular multiplication of a[0..8] & b[0..8],<br /><br />school book multiplication equipped with Montgomery reduction technique<br /><br />is used, which is why a[0..8], b[0..8] are expected to be in Montgomery form,<br /><br />while computed c[0..8] will also be in Montgomery form.<br /><br />See https://github.com/itzmeanjan/secp256k1/blob/6e5e654823a073add7d62b21ed88e9de9bb06869/field/base_field_utils.py#L101-L222 |
```

Comment:
note for the future: I think Bobbin commented in another PR, but we usually have the order reversed with the least-significant value deepest in the stack. In that case, the expected stack would be `[a7, a6, a5, ..., a0, b7, ..., b0,...]` etc. Can you add a task item to the issue created for Bobbin's comment in the BLAKE3 PR to update this as well?